### ISSUE TITLE
\hideinitializer doesn't work for new-style typedefs (using) (Origin: bugzilla #784809)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#6230]
status NEW severity *normal* in component *general* for *---*
Reported in version *1.8.13* on platform *Other*
Assigned to: Dimitri van Heesch

On 2017-07-11 17:01:02 +0000, h2+bugs@fsfe.org wrote:

    Pretty much what the title says.

**Comment by albert-github** [comment]
Creating a small example:
```
/// \file

/// \brief the using version
using MyInt_u = int;

/// \brief the using version hide
/// \hideinitializer
using MyInt_u_hide = int;
```

With the doxygen version 1.9.8 we see:
![image](https://github.com/doxygen/doxygen/assets/5223533/f434eb65-1ce5-43e0-95f0-5ef8ae13f30a)


and with the current master version (1.10.0 (4f6d3abebcfab7f8811d08a8f6ef00d1ef9feca3)) we see:
![image](https://github.com/doxygen/doxygen/assets/5223533/4cda9bb7-95f0-46b9-9b1e-ab43495a6bc7)

So with the current master version the initializer part is gone with the `hide` usings.

I think the problem is fixed but not released

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.10.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
